test/fixtures/members.yml

```diff
@@ -368,6 +368,48 @@ group_eleven_member_user_26:
   namespace_id: <%= ActiveRecord::FixtureSet.identify(:group_eleven) %>
   access_level: <%= Member::AccessLevel::OWNER %>
 
+group_twelve_member_john_doe:
+  user_id: <%= ActiveRecord::FixtureSet.identify(:john_doe) %>
+  created_by_id: <%= ActiveRecord::FixtureSet.identify(:john_doe) %>
+  namespace_id: <%= ActiveRecord::FixtureSet.identify(:group_twelve) %>
+  access_level: <%= Member::AccessLevel::OWNER %>
+
+subgroup_twelve_a_member_john_doe:
```

Comment:
Since the sub groups/projects under  group_twelve all have the same parent ancestor, you only need to have the john doe added as a member to the top level group and the subgroups and projects will inherit the membership. Only if you wanted to give the member a higher access level then they have in the parent group would you need to add a direct member to the subgroups/projects

Reply:
Makes sense! Fixed in 3a93671162a425316846ea3c51595790644eceff